dataclients/kubernetes/definitions/ingressvalidator.go

```diff
@@ -0,0 +1,68 @@
+package definitions
+
+import (
+	"fmt"
+
+	"github.com/zalando/skipper/eskip"
+)
+
+const (
+	skipperfilterAnnotationKey    = "zalando.org/skipper-filter"
+	skipperpredicateAnnotationKey = "zalando.org/skipper-predicate"
+	skipperRoutesAnnotationKey    = "zalando.org/skipper-routes"
+)
+
+type IngressV1Validator struct{}
+
+var defaultIngressV1Validator = IngressV1Validator{}
+
+func ValidateIngressV1(item *IngressV1Item) error {
```

Comment:
No tests added in skipper for now since we don't validate there, will try to add some.